pallets/manta-pay/src/benchmark/mod.rs

```diff
@@ -28,16 +23,29 @@ use manta_support::manta_pay::{
 };
 
 use manta_primitives::{
-    assets::{AssetConfig, AssetRegistry, FungibleLedger, TestingDefault},
+    assets::{AssetConfig, FungibleLedger},
     constants::TEST_DEFAULT_ASSET_ED,
     types::Balance,
 };
 use scale_codec::Decode;
 
-mod precomputed_coins;
+// mod precomputed_coins;
```

Comment:
commented code

Reply:
Should i delete this `precomputed_coins.rs` file? If this pr gets merged, seems this file is unnecessary.

Reply:
yeah seems like it